### DISCUSSION TITLE
Billing User Type Not Showing

### DISCUSSION THREAD

**Discussion by lbebels** [vercel/community#3136] [Help] [answered]
### Summary

Despite having a pro account, I am unable to add a Billing user to the team. I am only seeing Owner and Member types.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/30048710/6b147717-d7ec-4ab1-ad7d-2fbabe23629a)
![image](https://github.com/vercel/community/assets/30048710/b24b34b3-70f5-44e6-8120-1bb46ba63190)

**Comment by leerob** [comment, marked as answer]
You can now add a free team seat for managing billing on Pro: https://vercel.com/changelog/billing-role-on-pro
